.github/workflows/woke.yml

```diff
@@ -7,16 +7,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: 'Checkout'
-        uses: actions/checkout@v2
-
-      - uses: jitterbit/get-changed-files@v1
-        id: files
+        uses: actions/checkout@v4
+        
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v40
+        with:
+          write_output_files: true
+          files: |
+            **
 
       - name: 'woke'
+        if: steps.changed-files.outputs.any_changed == 'true'
```

Comment:
is this patch already on the main branch?

Reply:
ah looks like it is

Reply:
Yep, for @HEAD https://github.com/tektoncd/pipeline/blob/main/.github/workflows/woke.yml#L21:
```
if: steps.changed-files.outputs.any_changed == 'true'
```